src/content/headerFooterContent.js

```
@@ -11,29 +11,31 @@ chrome.runtime.onMessage.addListener(function (request, sender, sendResponse) {
+        case 'unloadFooterHTML': {
+            let footerHTMLCode = document.querySelector('textarea[name="footer"]').value || "";
+            logDebug("Unloading footer code", footerHTMLCode);
+            sendResponse({ code: footerHTMLCode });
             break;
+        }
 
-        case 'loadFooter':
-            logDebug("Loading footer code");
-            document.querySelector('#footer_content').value = request.code;
+        case 'loadFooterHTML': {
+            let footerHTMLCode = request.code;
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'footerHTMLCode' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)